improver/psychrometric_calculations/psychrometric_calculations.py

```diff
@@ -131,6 +131,58 @@ def calculate_svp_in_air(temperature: ndarray, pressure: ndarray) -> ndarray:
     return svp * correction.astype(np.float32)
 
 
+# Kappa is R/Cp,
+# where R is the gas constant in J kg-1 K-1
+# and Cp is the specific heat of dry air in J kg-1 K
+KAPPA = 287.0 / 1004.5
```

Comment:
Since both R and CP are defined in constants, I've just changed the code to use them. Why use another random Greek letter?

The value of CP in constants is slightly different that the one I used, which came from the GPP CDP code. Makes a small difference to the unit tests, but probably very little difference overall.